Fantasy Freaks/WinStat.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fantasy_Freaks {
+    internal class WinStat {
+        public static WinStat instance;
+
+        public WinStat() {
+            instance = this;
+        }
+
+        public int Week { get; set; }
+        public string OffName { get; set; }
+        public int OffScore { get; set; }
+        public int LowOffscore { get; set; }
+        public int HighOffscore { get; set; }
+        public string DefName { get; set; }
+        public int DefScore { get; set; }
+
+        public int eventDay() {
+            Random rnd = new Random();
+            int num = rnd.Next(0, 5);
+            return num;
+        }
+
+        public int Tpscore(int tp) {
```

Comment:
Can Tpscore have a more specific name, it's returning something in WinStat so it seems to be important, same with TYscore

Reply:
TPscore is for the total points the defending team allowed. It is passed in the number of points allowed from our data and then returns their "score" which will then be added to their total in winStat generator.

Reply:
can we rename it to CalculatePointsAllowed or something similar to that? tp should also be changed to TotalUserScoredPoints or something similar. The method name should make it clear what is being returned and the parameter names should make it clear what should be passed in without reading the code. C# Methods also start with a capital letter and the verb should come first "defScoreCalc" -> "CalculateDefensiveScore"

Reply:
minor stuff like renaming varaibles/methods can be annoying but with 5 people working on the project we'll need to make the code pretty readable so we don't have to ask questions everytime we look at other people's code

Reply:
For sure. Will work to edit the names for the methods and parameters so they are easier to follow

Reply:
![image](https://user-images.githubusercontent.com/58701759/202531677-b92fe73c-e35c-4050-b448-a46d053fdc98.png)


I renamed the methods and parameters, how does this look? Leaving the int arrays untouched as I am unsure how we will access the database. The parameter that is passed through the array (this can change) is used to pass the player or team's ID/string so it knows where to get that data.

Reply:
looks great 👍